x-pack/legacy/plugins/reporting/export_types/printable_pdf/types.d.ts

```diff
@@ -17,6 +17,18 @@ export interface JobParamsPDF {
   layout: LayoutInstance;
```

Comment:
This should not have `relativeUrl`: knowing that helps `relativeUrl: undefined` in `JobDocPayloadPDF` make more sense. That is a separate issue, added from https://github.com/elastic/kibana/pull/44130